src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -310,6 +310,8 @@
   - name: --create-cert
     short-summary: Create a self-signed certificate to use for the credential.
     long-summary: Use with `--keyvault` to create the certificate in Key Vault. Otherwise, a certificate will be created locally.
+  - name: --password -p
```

Comment:
New things?

Reply:
Just missing help text that I'm adding. :) Not sure why it wasn't getting flagged before.